hack/setup/quick-install/kserve-knative-mode-full-install-with-manifests.sh

```
@@ -991,10 +986,10 @@ install_kustomize() {
 
     log_info "Installing Kustomize ${KUSTOMIZE_VERSION} for ${os}/${arch}..."
 
-    if [[ -x "${BIN_DIR}/kustomize" ]]; then
-        local current_version=$("${BIN_DIR}/kustomize" version 2>/dev/null | awk 'match($0, /v[0-9.]+/) {print substr($0, RSTART, RLENGTH)}')
+    if command -v kustomize &>/dev/null; then
+        local current_version=$(kustomize version --short 2>/dev/null | grep -oP 'v[0-9.]+')
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)